src/main/resources/data/plasmid/lang/fr_fr.json

```diff
@@ -0,0 +1,38 @@
+{
+  "text.plasmid.game.started.player": "%s a démarré le jeu !",
+  "text.plasmid.game.stopped.player": "%s a arrêté le jeu !",
+  "text.plasmid.game.stopped.error": "Une erreur inattendue est survenu lors de l'arrêt du jeu !",
+  "text.plasmid.game.list": "Jeux enregistrés :",
+  "text.plasmid.game.channel.create": "Canal '%s' créé",
+  "text.plasmid.game.channel.remove": "Canal '%s' supprimé",
+  "text.plasmid.game.channel.connect.entity": "'%1$s' connecté à '%2$s''",
+  "text.plasmid.game.channel.connect.block": "'%1$s' connecté au bloc avec pour position (%2$s; %3$s; %4$s)",
+  "text.plasmid.game.join": "%s a rejoint le lobby !",
+  "text.plasmid.game.join.error": "Une erreur inattendue est survenue lors de la connection au jeu !",
+  "text.plasmid.game.open.error": "Une erreur inattendue est survenue lorsque que le jeu démarrait !",
+  "text.plasmid.game.open.opened": "%1$s a ouvert %2$s! ",
+  "text.plasmid.game.open.join": "Cliquez ici pour rejoindre",
+  "text.plasmid.game.waiting_lobby.bar.waiting": "En attente de joueurs...",
+  "text.plasmid.game.waiting_lobby.bar.countdown": "Démarre dans %s secondes !",
+  "text.plasmid.game.waiting_lobby.bar.cancel": "Démarrage annulé ! ",
```

Comment:
```suggestion
  "text.plasmid.game.waiting_lobby.bar.cancel": "Démarrage annulé !",
```

Reply:
this space is needed, it's in the english translation too.